aosp_diff/base_aaos/packages/modules/Bluetooth/0001-Fix-metadata-issue-when-reconnect-pixel.patch

```diff
@@ -0,0 +1,46 @@
+From da592843a355924677f877a2e6013e58bec83800 Mon Sep 17 00:00:00 2001
+From: "Bhadouria, Aman" <aman.bhadouria@intel.com>
+Date: Mon, 25 Nov 2024 04:27:53 +0000
+Subject: [PATCH] Fix metadata issue when reconnect pixel
+
+When reconnecting pixel some times metadata is not synced.
+
+Issue: bt-stack issuing two AVRC browse channel, one as acceptor,
+other as initator, which is causing collision in lcid and resulting
+in l2cap disconnection of avrc browse channel, resulting in metadata
+sync issue.
+
+Solution: Don't initiate avrc browse channel as initiator, open only
+as acceptor. Now browse channel is always success and metadata is
+seen everytime.
+
+Tracked-On: OAM-127243
+Signed-off-by: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
+Signed-off-by: Bhadouria, Aman <aman.bhadouria@intel.com>
+---
+ system/bta/av/bta_av_act.cc | 9 +++++++--
+ 1 file changed, 7 insertions(+), 2 deletions(-)
+
+diff --git a/system/bta/av/bta_av_act.cc b/system/bta/av/bta_av_act.cc
+index e94c26d51f..17c728f11f 100644
+--- a/system/bta/av/bta_av_act.cc
++++ b/system/bta/av/bta_av_act.cc
+@@ -664,8 +664,13 @@ void bta_av_rc_opened(tBTA_AV_CB* p_cb, tBTA_AV_DATA* p_data) {
+   if ((p_cb->features & BTA_AV_FEAT_BROWSE) &&
+       (rc_open.peer_features & BTA_AV_FEAT_BROWSE) &&
+       ((p_cb->rcb[i].status & BTA_AV_RC_ROLE_MASK) == BTA_AV_RC_ROLE_INT)) {
+-    log::verbose("opening AVRC Browse channel");
+-    AVRC_OpenBrowse(p_data->rc_conn_chg.handle, AVCT_INT);
++      log::verbose("Not opening AVRC Browse channel");
```

Comment:
Indentation issue, pls make sure all added lines are equally intended.

Reply:
Change the verbose logging as
"Not opening AVRC Browse channel as INT"

Please refer the exact logging we did in older patches.